src/kani-compiler/cbmc/src/goto_program/location.rs

```diff
@@ -13,7 +13,14 @@ pub enum Location {
     BuiltinFunction { function_name: InternedString, line: Option<u64> },
     /// Location in user code.
     /// `function` is `None` for global, `Some(function_name)` for function local.
-    Loc { file: InternedString, function: Option<InternedString>, line: u64, col: Option<u64> },
+    Loc {
```

Comment:
So if I'm understanding it correctly, I should add a new variant for Location called Stmt::Assert instead of using Loc (local functions) to store information about property_class and comment?

Reply:
No, what I meant is to add new a property class field to the new `StmtBody::Assert` variant (in `cbmc/src/goto_program/stmt.rs`).

Reply:
That seems a much better solution, will make those changes.

Reply:
Isn't this crate trying to match what CBMC is doing though? Also, the `propery_class` might be relevant at least for cover statements, right?

Reply:
It is trying to match CBMC's structure but maybe we only need to map it at the end when converting to Irep's